vsce/server/src/server.ts

```diff
@@ -342,11 +342,22 @@ connection.onDidChangeWatchedFiles(_change => {
 });
 
 export interface ErrorLocation {
+	code: string;
 	range: Range;
 	errorMessage: string; // e.g. id ref: Invalid unbound identifier: declassify.
 	suggestions: string[]; // e.g. literally this whole thing: "declassify","array","assert","assume","closeTo"
 }
 
+// Based on
+// https://github.com/reach-sh/reach-lang/blob/master/hs/src/Reach/AST/Base.hs#L84
+type ReachCompilerErrorJSON {
+	ce_position: number[];
+	ce_suggestions: string[];
+	ce_errorMessage: string;
+	ce_offendingToken?: string;
```

Comment:
```suggestion
	ce_offendingToken: string | null;
```
This field will always be in the error JSON, and it is nullable.

Reply:
I'm glad you're a reviewer. Thank you!